torch/onnx/symbolic.py

```diff
@@ -562,12 +564,40 @@ def clamp(g, self, min, max):
     return g.op("Clip", self, min_f=min, max_f=max)
 
 
-def max(g, self, other):
-    return g.op("Max", self, other)
-
-
-def min(g, self, other):
-    return g.op("Min", self, other)
+# torch.max (same for torch.min) actually has two interfaces smashed together:
+# torch.max(x, dim, keepdim) and torch.max(x, y)
```

Comment:
CC @colesbury; a long time ago, we had a discussion where I got the impression we were trying to avoid overloads like this. Is `max` not following the pattern?

Reply:
Yeah, we should split `max`. Tracked here: https://github.com/pytorch/pytorch/issues/2739. (Updating the issue now)

We need to decide if `torch.max(x, y)` should become `torch.fmax(x, y)` or `torch.maximum(x, y)`. The difference is how NaNs are handled: `fmax` prefers non-NaN while `maximum` prefers NaNs. (Our current behavior is to prefer the value from `x`, which I don't think is worth keeping).

https://docs.scipy.org/doc/numpy-dev/reference/generated/numpy.fmax.html#numpy.fmax
https://docs.scipy.org/doc/numpy-dev/reference/generated/numpy.maximum.html

We can keep backwards compatibility for `max(x, y)` by a